bitnami/keycloak/templates/statefulset.yaml

```
@@ -41,6 +41,7 @@ spec:
         {{- end }}
     spec:
       serviceAccountName: {{ template "keycloak.serviceAccountName" . }}
+      automountServiceAccountToken: {{ .Values.automountServiceAccountToken }}
```

The value used is wrong value, it should be `.Values.serviceAccount.automountServiceAccountToken`
```suggestion
      automountServiceAccountToken: {{ .Values.serviceAccount.automountServiceAccountToken }}
```

---

I've added the option to dis-/en-able it in the pod and the ServiceAccount.
This adds the possibility to disable the service Account and also disable automounting the default Service Account token.